config/application.rb

```diff
@@ -56,6 +56,9 @@ class Application < Rails::Application
       config.session_store :feeder_active_record_store, key: "_feeder_session"
     end
 
+    # Version the cache by the application version
+    ENV["RAILS_APP_VERSION"] = VERSION
```

Comment:
We need to start cycling our redis cache on deploys, when the representers have changed.  This adds the app version (`1.0.0` at the moment) to the redis key.